src/content/self-hosted/administration/configuration.mdx

```diff
@@ -1010,6 +1010,33 @@ wildcardCertificate:
       key: "ca.crt"
 ```
 
+### user
+
+#### serviceAccount
+
+The configured annotations and labels will be applied to each service account created by Okteto.
+These annotations and labels are additional to the ones already applied by Okteto.
+
+- `annotations`: Annotations applied to each service account generated by Okteto for user accounts.
+- `labels`: Labels applied to each service account generated by Okteto for user accounts.
+
+```yaml
+user:
+  serviceAccount:
+    annotations:
+      custom.annotation/one: one
+      custom.annotation/two: two
+    labels:
+      custom.label/one: one
+      custom.label/two: two
+```
+
+A common use case for custom service account annotations is to get access via IAM Roles or Workload Identity to create Cloud resources as part of your Okteto Manifest `deploy` commands.
```

Comment:
Can we add links to our docs on IAM Roles and Workload Identity? (even if they are on the community site)
```suggestion
A typical use case for this is integrating with IAM Roles or Workload Identity to create Cloud resources as part of your Okteto Manifest `deploy` commands.
```

Reply:
you mean this reply ? https://community.okteto.com/t/support-for-gke-workload-identities-for-okteto-self-hosted/39/4

Reply:
i've reworded this sentence and added the link under `get access via IAM Roles or Workload Identity`